docs/ibc/1_IBC_ARCHITECTURE.md

```diff
@@ -0,0 +1,141 @@
+# 1: IBC Architecture
+
+> This is an overview of the high-level architecture & dataflow of the IBC protocol.
+
+> For a broad set of protocol design principles, see [here](./2_IBC_DESIGN_PRINCIPLES.md).
+
+> For definitions of terms used in IBC specifications, see [here](./3_IBC_TERMINOLOGY.md).
+
+This document outlines the architecture of the authentication, transport, and ordering layers of the IBC protocol stack. This document does not describe specific protocol details — those are contained in individual ICSs.
+
+## What is IBC?
+
+The *inter-blockchain communication protocol* is designed for use as a reliable, authenticated module-to-module communication protocol between modules running on independent distributed ledgers across an untrusted network layer.
```

Comment:
Maybe you can say: "The IBC is reliable and secure inter-module communication protocol, where modules run on independent distributed ledgers on top of untrusted network later".

Reply:
Updated.